### ISSUE TITLE
Installing Warp with adding APT repo/key manually (without need to download .deb)

### ISSUE THREAD

**Issue by DWolf-19** [warpdotdev/Warp#4516]
### Describe the solution you'd like?

It's much more convenient for me to type a few commands into terminal instead of downloading .deb manually. 

Something like this could be added to the docs:
```bash
curl -fsSL https://repo.example.com/gpg | apt-key add -

add-apt-repository "deb [arch=$(dpkg --print-architecture)] \
    https://repo.example.com/ \
    $(. /etc/os-release && echo "$VERSION_CODENAME") stable"

apt install warp-terminal
```

P.S. Also actively waiting for flatpak.

### Additional context

OS: I'm using Ubuntu 22.04

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Thanks, we'll have to look into this and create the equivalent for Fedora/Arch/Suse, etc. maybe in it's own section of the installation/update docs.

**Comment by pacificera** [comment]
I've decided to stop using warp until there is a flatpak installation.  Updating with an rpm file on fedora silverblue workstations is too painful for an application that has frequent updates like warp.  I have to rpm-ostree remove it, reboot, and rpm-ostree install the rpm file, and reboot again every time there is a warp update.  Too painful!

**Comment by DWolf-19** [comment]
Yeah, I'm not using it right now, waiting for flatpak. In addition, this is a trashing of the home directory with Warp's configs and caches. I want everything to be neatly folded in `~/.var/app/dev.warp.Warp`.

P.S. _It seems that we have moved away from the issue topic._

**Comment by dannyneira** [comment]
yes @pacificera please open a new request for flatpak support, FWIW, there is an appimage option atm, which may be easier for install/upgrade. https://docs.warp.dev/getting-started/getting-started-with-warp#appimage

**Comment by dannyneira** [comment]
@DWolf-19 @pacificera We'll be tracking Flatpak support separately here: https://github.com/warpdotdev/Warp/issues/4662

**Comment by KaKi87** [comment]
Hello,
Why was this issue closed ?
Has an APT repo been created ? If yes I'm sorry I missed it.
Thanks

**Comment by KaKi87** [comment]
Oh, thanks.

That's not very visible though, it would be nice to have the option on the website dropdown and to use sections instead of tabs in the docs.
